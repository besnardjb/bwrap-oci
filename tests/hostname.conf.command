#!/bin/sh
ocitools generate --hostname=atomic
