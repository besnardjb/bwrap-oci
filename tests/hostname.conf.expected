bwrap --bind rootfs / --unshare-pid --unshare-net --unshare-ipc --unshare-uts --proc /proc --tmpfs /dev --tmpfs /dev/shm --mqueue /dev/mqueue --chdir / --setenv PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin --setenv TERM xterm --uid 0 --gid 0 sh
